acinclude/win32-sspi.m4

```diff
@@ -0,0 +1,38 @@
+## Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+# check if the Windows SSPI requirements are available and work.
+# first argument is the variable containing the result
+#   (will be set to "yes" or "no")
+# second argument is the commands to run on success
+#
+AC_DEFUN([SQUID_CHECK_WIN32_SSPI],[
+  AC_CHECK_HEADERS([w32api/windows.h windows.h],[
+    squid_cv_win32_sspi=yes
```

Comment:
If possible, please move this assignment inside the second AC_CHECK_HEADERS(). I assume the code "works" as is, but this assignment looks like the wrong "default" here IMHO.

Reply:
Right, I only looked at the lines below the `yes` assignment, so my "second" AC_CHECK_HEADERS is your "third" -- we are talking about the same AC_CHECK_HEADERS() line that is described in PR code as "required headers".

> the if-found and if-not-found cases get run for each of them.

Ah, I did not realize that! I consider this change request addressed, thank you.